mne/annotations.py

```diff
@@ -977,3 +1030,59 @@ def events_from_annotations(raw, event_id="auto",
                 (list(event_id_.keys()),))
 
     return events, event_id_
+
+
+@verbose
+def annotations_from_events(events, sfreq, event_desc=None, first_samp=0,
+                            orig_time=None, verbose=None):
+    """Convert an event array to an Annotations object.
+
+    Parameters
+    ----------
+    events : ndarray, shape (n_events, 3)
+        The events.
+    sfreq : float
+        Sampling frequency.
+    event_desc : dict | list | 1darray | callable | None
```

Comment:
Instead of `1darray`, we use `ndarray, shape (n_events,)` or something along these lines (same below for `**1darray**`).

Maybe we could use list-like here instead?

Reply:
I thought about using `ndarray, shape (n_events_ids,)` or similar, but it felt unnecessarily long and not much clearer than `1darray` ... ?

`list-like` is fine IMO, simple and clear

Reply:
usually in these cases we just say `array-like`, you can specify the shape if you want or just explain it in the description